### DISCUSSION TITLE
which approch is more suitable?

### DISCUSSION THREAD

**Discussion by jaimeenkoffeekodes** [vercel/next.js#64598] [Help]
### Summary

![2212](https://github.com/vercel/next.js/assets/149799052/a7acd94e-1a6d-4bca-8e65-3ffeb0971e06)
![code](https://github.com/vercel/next.js/assets/149799052/76e606a6-f8d0-43dc-be8e-097a8e178d43)

**Comment by ananuness** [comment]
The first option, also it would be even better if you separated the `FormikContext`, `BreadCrumbDeliveryNote` and `Page` into different files so as not to have this excess information in a single file.

**Reply by jaimeenkoffeekodes** [reply]
thanks ana!
